src/normalize/entities.ts

```diff
@@ -98,6 +99,7 @@ const commonMessageFields = {
 const commonInviteFields = {
 	compNum: 'componentNum',
 	calItemId: 'calendarItemId',
+	ciFolder: 'ciFolder',
```

Comment:
If this is not being renamed it can be left out.

Reply:
removed